### PR TITLE
style: bug fixes and tweaks

### PR DESCRIPTION
**Pull request by chelliah**

### CODE REVIEW

File: src/editor-actions/FontColor.js [hunk 1]
```diff
@@ -51,7 +57,7 @@ export default class FontColor extends React.Component {
       animationFillMode: 'both'
     };
     if (!hasRoomToRenderBelow) {
-      dropdownStyles.bottom = dropdownStyles.top + 55;
+      dropdownStyles.bottom = dropdownStyles.top;
```
Comment (GuruM) [inline src/editor-actions/FontColor.js]: This means that the font color option box might cover up the text its editing, right?

File: src/editor-actions/ImageUploadWithPresets.js [hunk 2]
```diff
@@ -286,6 +286,8 @@ export default class ImageUploadWithPresets extends React.Component {
 
     const urlWithoutProtocol = url.replace(/^https?\:\/\//i, "//");
 
+    console.log(urlWithoutProtocol);
```
Comment (GuruM) [inline src/editor-actions/ImageUploadWithPresets.js]: logging

**Comment by GuruM** [comment]
It may seem over the top, but I'd recommend splitting up each independent change into separate PRs. Many of the merged PRs in this repo are 1-liners and I generally submitted at the same time; just makes it easier to track down problems and identify what changes are being made from the repo's PR list alone

File: src/editor-actions/FontColor.js (hunk 1 above)
Comment (chelliah) [inline src/editor-actions/FontColor.js]: ahh yeah I can add that back in if you want. When I was testing the wysiwyg it felt really out of place to have it so detached (Also only guarantees you will clear a line or so of text) .  Might be better to use opacity or something

**Comment by chelliah** [comment]
For sure, I tried to make up for the generality of the pr by including pretty specific/clear commit messages. I'll keep that in mind for the future. I can split these up but it'd take a little bit of fighting with git

File: src/editor-actions/FontColor.js (hunk 1 above)
Comment (chelliah) [inline src/editor-actions/FontColor.js]: actually i'll add this back in, will try experimenting with some different options though

**Pull request merged**